db/upgrade.php

```diff
@@ -500,6 +500,58 @@ function xmldb_plagiarism_turnitin_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2021081301, 'plagiarism', 'turnitin');
     }
 
+    if ($oldversion < 2021101800) {
```

Comment:
ah, this is going to be a royal pita if and when TII decide to merge this upstream because we will have forked the version numbers. So we want to make these version bumps as small as possible, possibly even fractional micro bumps so the chance of the version getting further ahead of what the upstream version is, is as small as possible

Reply:
I updated the version to `2021091502` which is just minimized bump.
I manually cherry-picked 2 commits and added comments.

Reply:
fyi (if it's useful) when I run into this sort of scenario I tend to write a custom script to fix things up rather than bumping the plugins own version.php - that way it doesn't touch the core code - I just worry that any local bump on version.php could result in an upstream upgrade step being missed.